lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -91,16 +71,14 @@ namespace :webpack do
     end
   end
 
-  task :paths => 'ui:load_inflectors' do
-    paths_config = ManageIQ::UI::Classic::Engine.root.join('config/webpack/paths.json')
+  task :paths do
+    json = JSON.pretty_generate(
+      :info    => "This file is autogenerated by rake webpack:paths, do not edit",
+      :output  => Rails.root.to_s,
+      :engines => asset_engines.map { |p| [p.webpack_name, p.path] }.to_h
+    ) << "\n"
 
-    File.open(paths_config, 'w') do |file|
-      file.write(JSON.pretty_generate(
-        :info    => "This file is autogenerated by rake webpack:paths, do not edit",
-        :output  => Rails.root.to_s,
-        :engines => asset_engines_plugins_hash,
-      ) + "\n")
-    end
+    File.write(ManageIQ::UI::Classic::Engine.root.join('config/webpack/paths.json'), json)
```

Comment:
Seems like a bit of a pedantic change (`File.write` from `File.open`), but okay.

Reply:
Can we keep the `+ "\n"` bit please? Otherwise, this is a file without the last EOL, which is annoying when catting.

Reply:
Yeah, it's pedantic, but I never really ever liked the `.open(.., "w" (or "rw" or "a", etc))` interface vs the separate `.read` and `.write`

Reply:
Yeah, that comment specifically you could have assumed a large helping of :trollface: emojis